docs/deployment/architecture.md

```diff
@@ -1,13 +1,7 @@
 # Architecture
 
-We use docker-compose to orchestrate containers.
+We use docker-compose to orchestrate containers with a reverse proxy that manages all incoming traffic to the service. The database and LiteLLM proxy are only accessed by the core app.
 
 <p align="center">
-  <img src="../../images/architecture-docker.png" alt="Architecture"/>
-</p>
-
-A reverse proxy manages all incoming traffic to the service. The database is only accessed by the core app.
-
-<p align="center">
-  <img src="../../images/architecture-traffic.png" alt="Flow"/>
+  <img src="../../images/architecture.png" alt="Flow"/>
```

Comment:
/nit Can you move the "LiteLLM Proxy" icon down to be in a similar location as Postgres?